Dockerfile

```diff
@@ -35,6 +35,9 @@ RUN pip install -r requirements.txt
 COPY test_requirements.txt ${HOME}/
 RUN pip install -r test_requirements.txt
 
+# HACK: install custom version of glean parser
+RUN pip install -U git+https://github.com/mozilla/glean_parser@tags
```

Comment:
Obviously this needs to be removed before we merge. Instead we should version bump glean_parser, release a new version, and have probe_scraper depend on it.